external/webserver/http_client.c

```diff
@@ -32,6 +32,8 @@
 #include "http_log.h"
 
 #define MIN_WS_HEADER_FIELD 2
+#define MAX_CLIENT_REQUEST 999999
```

Comment:
How about to describe why MAX_CLIENT_REQUEST was set to 999999

Reply:
Then please add above your comment to the code for better understand.

Reply:
done